### DISCUSSION TITLE
How to prevent jotai-devtools/styles.css from overwriting my Mantine theme?

### DISCUSSION THREAD

**Discussion by npearson72** [jotaijs/jotai-devtools#166] [Q&A] [answered]
I'm using Mantine 7.12.1. I have my own `index.css` that I used to define global styling and the Jotai DevTools styles.css file is overwriting them.

In particular, my `p` definition is being overwritten.

Is there a way to prevent this?

**Comment by arjunvegda** [comment, marked as answer]
hey @npearson72 

Thanks for reaching out! This is a known issue. We have [a PR](https://github.com/jotaijs/jotai-devtools/pull/157) open to fix this in the short term but are blocked by its side effect.

In the long run, we're working on moving away from Mantine altogether. Sorry for the inconvenience!

**Reply by npearson72** [reply]
Thanks @arjunvegda 

@rtivital any way to bring back nested to `MantineProvider` in 7x to allow passing library specific themes that won't override consumer code?
